GNImageProgressBar/GNImageProgressBar.swift

```diff
@@ -41,6 +43,18 @@ public class GNImageProgressBar: UIView {
     
     public var fixedSize: CGSize = .zero
     
+    public init(image: UIImage, size: CGSize, backgroundImage: UIImage, backgroundImageAlpha: CGFloat = 1, shouldShowBackgroundImage: Bool = true, fillDirection: FillDirection, progress: CGFloat = 0) {
+        self.image = image
+        self.backgroundImage = backgroundImage
+        self.size = size
+        self.shouldShowBackgroundImage = shouldShowBackgroundImage
+        self.backgroundImageAlpha = shouldShowBackgroundImage ? backgroundImageAlpha : 0
+        self.fillDirection = fillDirection
+        self.progress = progress
+        
+        super.init(frame: .zero)
+        setup()
+    }
```

Comment:
Since the shouldShowBackgroundImage field is public var, I think it makes no sense to remove this field from my initializer. So I'll just add bgImage to the existing one. Thanks for the comments.